internal/manifests/datasource/grafana.go

```diff
@@ -0,0 +1,38 @@
+package datasource
```

Comment:
```suggestion
package grafana
```

Can we rename this package (and directory) to `grafana` and this file to `datasource.go`?